tools/clang/test/hlsl_cmds_test/NonUniform.test

```diff
@@ -0,0 +1,56 @@
+// RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
```

Comment:
Am I correct that these are all copied without alteration from the utils directory? Is there any need for the old versions? Were there any changes?

I see that some were renamed from .hlsl to .test, but some were not. Why is that?

Reply:
I changed hlsl to test because not all hlsl files have a test use it directly.
I'll move all hlsl into Input directory and keep the test as only lit command as Chris suggested.

We don't need the old version once the lit version is enabled by default.
There should no change on the test itself.

Reply:
If the files are actual HLSL source we shouldn't be renaming them to `.test`